### ISSUE TITLE
How to set background-image?

### ISSUE THREAD

**Issue by pkufool** [jaywcjlove/markdown-to-html-cli#57]
A nice tool, Thanks!

My question is, how to set background-image for the whole page, thanks!

**Comment by jaywcjlove** [comment]
Oh, that's a good question, I should have provided a `--css` parameter to change the default style.
@pkufool

**Comment by pkufool** [comment]
> Oh, that's a good question, I should have provided a `--css` parameter to change the default style. @pkufool

Yes, I tried adding the style to the package.json (even using  `!important`), but it didn't take effect.

**Comment by jaywcjlove** [comment]
@pkufool Upgrade `3.5.0`

```bash
npx markdown-to-html-cli --source README.md --style=./style.css
npx markdown-to-html-cli --source README.md --style='body { color: red; }'
```

**Comment by pkufool** [comment]
Wow, You are so fast! Thank you! I will try it this evening.
